sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/AdministratorContact.java

```diff
@@ -52,7 +52,7 @@ public Administrator(String firstName, String lastName, String email) {
      * @param email the email of the admin.
      * @param contact tne contact info of the admin.
      */
-    public Administrator(String firstName, String lastName, String email, String contact) {
+    public AdministratorContact(String firstName, String lastName, String email, String contact) {
```

Comment:
What is "contact"? We have "phone" in .NET, which matches the REST model: https://docs.microsoft.com/en-us/rest/api/keyvault/getcertificateissuer/getcertificateissuer#administratordetails